Netkan/Sources/Github/GithubRelease.cs

```diff
@@ -1,21 +1,20 @@
 using System;
+using System.Collections.Generic;
 using CKAN.Versioning;
 
 namespace CKAN.NetKAN.Sources.Github
 {
     public sealed class GithubRelease
     {
-        public string           Author       { get; }
-        public ModuleVersion    Version      { get; }
-        public Uri              Download     { get; }
-        public DateTime?        AssetUpdated { get; }
+        public string        Author { get; }
+        public ModuleVersion Tag    { get; }
+        public IEnumerable<GithubReleaseAsset> Assets { get; }
```

Comment:
Any objections to making this a `List`? `IEnumerable`'s potential for lazy evaluation makes me nervous when it's in a class.

Reply:
Nope, fine with that. Good idea, given that we've had some interesting experiences with `IEnuerables` in netkan. Only need to update a few arrays to lists.